modules/fscloud/variables.tf

```diff
@@ -3,50 +3,50 @@
 ##############################################################################
 
 variable "resource_group_id" {
-  description = "ID of resource group to use when creating the event stream instance"
+  description = "The resource group ID to use when creating the Event Streams instance."
   type        = string
 }
 
 variable "name" {
   type        = string
-  description = "The name to give the IBM Event Notification instance created by this module."
+  description = "The name of the Event Notifications instance that is created by this module."
 }
 
 variable "tags" {
   type        = list(string)
-  description = "Optional list of tags to be added to the Event Notification instance"
+  description = "The optional list of tags to add to the Event Notifications instance."
   default     = []
 }
 
 variable "region" {
   type        = string
-  description = "IBM Cloud region where event notification will be created, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney), eu-es (Madrid)"
+  description = "The IBM Cloud region where the Event Notifications resource is created. The following regions are supported: `us-south` (Dallas), `eu-gb` (London), `eu-de` (Frankfurt), `au-syd` (Sydney), `eu-es` (Madrid)"
   default     = "us-south"
 }
 
 variable "skip_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all event notification instances in the provided resource group reader access to the instance specified in the existing_kms_instance_guid variable."
+  description = "Set this variable value to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group reader access to the instance specified in the `existing_kms_instance_guid` variable."
   default     = false
 }
 
 variable "existing_kms_instance_crn" {
-  description = "The CRN of the Hyper Protect Crypto Services or Key Protect instance. To ensure compliance with FSCloud standards, it is required to use HPCS only."
+  description = "The Cloud Resource Name (CRN) of the Hyper Protect Crypto Services (HPCS) or Key Protect instance. To ensure compliance with IBM Cloud Framework for Financial Services (FS Cloud) standards, it is required to use HPCS only."
```

Comment:
I don't think HPCS is an approved acronym but you can check

Reply:
FS Cloud is definitely not. I'm not sure how consistent I have been in other repos for HPCS, but I don't think it's approved.